test/anvil/anvil-global-setup.ts

```diff
@@ -1,24 +1,71 @@
 import { CreateAnvilOptions, createAnvil } from '@viem/anvil';
 import { sleep } from '../lib/utils/promises';
 
+type NetworkSetup = {
+    rpcEnv: string;
+    fallBackRpc: string | undefined;
+    port: number;
+    forkBlockNumber: bigint;
+};
+
+type Networks = NetworkSetup[];
+
+const NETWORKS: Networks = [
```

Comment:
```suggestion
type NetworksWithFork = Extract<
    keyof typeof ChainId,
    'MAINNET' | 'POLYGON' | 'FANTOM'
>;

const NETWORKS: Record<NetworksWithFork, NetworkSetup> = {
    MAINNET: {
        rpcEnv: 'ETHEREUM_RPC_URL',
        fallBackRpc: 'https://cloudflare-eth.com',
        port: 8545,
        forkBlockNumber: 18043296n,
    },
    POLYGON: {
        rpcEnv: 'POLYGON_RPC_URL',
        // Public Polygon RPCs are usually unreliable
        fallBackRpc: undefined,
        port: 8137,
        // Note - this has to be >= highest blockNo used in tests
        forkBlockNumber: 44215395n,
    },
    FANTOM: {
        rpcEnv: 'FANTOM_RPC_URL',
        // Public Fantom RPCs are usually unreliable
        fallBackRpc: undefined,
        port: 8138,
        forkBlockNumber: 65313450n,
    },
};

export default async function () {
    for (const network of Object.values(NETWORKS)) {
        const config = getAnvilOptions(network);
        await startFork(config);
    }
}
```

Reply:
Instead of an array I would use an object that we can reference in other tests (see comments below)